app/basicComponents/DropDown.tsx

```diff
@@ -28,7 +28,7 @@ const Wrapper = styled.div<{
       },
       isLightSkin,
```

Comment:
A very confusing name.
As far as I understood this is some kind of alternative look (that is dark in "modern" and "dark" themes, but is not used in the "light" theme). So `isDarkMode` sounds better :)
Or we may call it `alternative` / `isDark` mb with suffix `Look` — then it will be pretty clear property name 🤔